src/olympia/amo/tests/test_utils_.py

```diff
@@ -172,3 +174,9 @@ def bar(self):
     del foo.bar
     assert foo.bar == 'original value'
     assert callme.call_count == 1
+
+
+@pytest.mark.parametrize('lang', settings.AMO_LANGUAGES)
+def test_get_locale_from_lang(lang):
+    """Make sure all languages in settings.AMO_LANGUAGES can be resolved."""
+    get_locale_from_lang(lang)
```

Comment:
Hmm, I'll see if I can find out how. I don't want to have another
mapping of AMO_LANGUAGES to Locale instances but maybe that's not a
bad idea...
 
 
On Thu, May 26, 2016, at 10:04 AM, Mathieu Pillard wrote:
> In src/olympia/amo/tests/test_utils_.py[1]:


> > @@ -172,3 +174,9 @@ def bar(self): >      del foo.bar >      assert
> > foo.bar == 'original value' >      assert callme.call_count == 1 > +
> > > + > +@pytest.mark.parametrize('lang', settings.AMO_LANGUAGES) >
> > +def test_get_locale_from_lang(lang): > +    """Make sure all
> > languages in settings.AMO_LANGUAGES can be resolved.""" > +
> > get_locale_from_lang(lang)
>
> Should we check the result as well while we're at it ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub[2]
>
>
 

Links:

  1. https://github.com/mozilla/addons-server/pull/2757#discussion_r64705274
  2. https://github.com/mozilla/addons-server/pull/2757/files/010ba0c7b21893f126dda81c1dfb74e36073e66e#r64705274